qiskit/pulse/library/discrete.py

```diff
@@ -28,14 +28,14 @@
 
 
 @deprecate_func(
-    since="0.25.0",
-    additional_msg="The discrete pulses library, including constant() is pending deprecation."
+    since="1.0.0",
```

Comment:
A headsup. The goal (at least, aspirational) is to release a 1.0 without deprecation warnings. 
```suggestion
    since="0.46.0",
```
If you introduce the deprecation in 0.46 (planned for week before Qiskit 1.0 and EoL until mid 2024), you can remove it in Qiskit 1.0.

Reply:
Nice. I missed that. Thanks!